src/api/entities/Account/MultiSig/index.ts

```diff
@@ -37,12 +39,32 @@ export class MultiSig extends Account {
    */
   public constructor(identifiers: UniqueIdentifiers, context: Context) {
     super(identifiers, context);
+    const { isV6 } = context;
+
     this.modify = createProcedureMethod(
       {
         getProcedureAndArgs: modifyArgs => [modifyMultiSig, { multiSig: this, ...modifyArgs }],
       },
       context
     );
+
+    if (isV6) {
+      this.joinCreator = createProcedureMethod(
+        {
+          getProcedureAndArgs: joinArgs => [joinCreator, { multiSig: this, ...joinArgs }],
+          optionalArgs: true,
+        },
+        context
+      );
+    } else {
+      this.joinCreator = function () {
```

Comment:
This should have a return type like GitHub says, even if it requires some weird cast

Reply:
Why not put this logic inside the procedure though? Then it would be just a single throw, vs having to declare some inline function